android/datastore.md

```diff
@@ -10,7 +10,7 @@ title: DataStore
 <br />
 
 **Note**
-Amplify Android is in preview mode and is not intended for production usage at this time. We welcome feedback to improve your experience in using Amplify Android.
+This guide shows how to build an app using our Amplify Libraries for Android (Preview) and the Amplify CLI toolchain.
```

Comment:
What was the reasoning behind this change (and the others like it) since it seems like it significantly weakens the warning about the nature of the current state of the library?

Reply:
The current state of library is preview. We are reflecting that state in the old and new message as well. The reasoning for change is first we should be directing customer to docs for products that are GA and not preview by default. This is a widely adopted AWS practice to direct to GA rather than preview. We did the other way around to get more feedback when we launched. Second, existing users should not get confused on state of SDKs as we work on Amplify which seemed to be the case earlier.  Hope that clarifies it more.